app/src/main/java/pm/gnosis/heimdall/ui/safe/pending/SafeCreationFundFragment.kt

```diff
@@ -87,10 +89,35 @@ class SafeCreationFundFragment : BaseFragment() {
     }
 
     private fun onCreationInfo(info: SafeCreationFundContract.CreationInfo) {
-        layout_pending_safe_address.text = info.safeAddress
-        info.paymentToken?.let {
-            layout_pending_safe_required_minimum_transfer_value.text = it.displayString(info.paymentAmount, roundingMode = RoundingMode.UP)
+
+        val addressString = SpannableStringBuilder(info.safeAddress)
+        addressString.setSpan(ForegroundColorSpan(context!!.getColorCompat(R.color.address_boundaries)), 0, 4, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+        addressString.setSpan(
+            ForegroundColorSpan(context!!.getColorCompat(R.color.address_boundaries)),
+            addressString.length - 4,
+            addressString.length,
+            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+        pending_safe_safe_address.text = addressString
+
+        pending_safe_safe_image.setAddress(info.safeAddress.asEthereumAddress())
+
+        info.tokenWithBalance?.let {
+            pending_safe_amount_needed.text = it.token.displayString(info.paymentAmount, roundingMode = RoundingMode.UP)
+
```

Comment:
is it ok in this project to have a blank line after each code line?

Reply:
more than one line is not ok. Here separating initialisation of different ui elements with empty line makes it more readable. Would you rather squash everything together to make it more compact?